### ISSUE TITLE
QUESTION: Theme won't build on netlify

### ISSUE THREAD

**Issue by tom-lex** [pacollins/hugo-future-imperfect-slim#166]
I’m trying to build a test site on netlify and am getting the following error:

`9:45:06 PM: Error: Error building site: failed to render pages: render of "taxonomyTerm" failed: "/opt/build/repo/layouts/_default/terms.html:10:29": execute of template failed: template: _default/terms.html:10:29: executing "main" at <.Page.Permalink>: can't evaluate field Page in type hugolib.OrderedTaxonomyEntry`

I’ve checked layouts/_default/terms.html for anything obvious, but I don’t see a thing.
The site is posted on [Gitlab](https://gitlab.com/LinuxExperienceXYZ/lexweb). 

In an effort to further troubleshoot this, I tried a new deploy with just the [example site data](https://gitlab.com/LinuxExperienceXYZ/vanilla). I did nothing to the hugo site files except install the theme, copy the example site files and init git, etc.

I did indeed get the exact same error. 

My build settings on netlify are:
```
### [Build settings](https://app.netlify.com/sites/zen-dijkstra-c0be9a/settings/deploys#build-settings)

Repository
[gitlab.com/LinuxExperienceXYZ/lexweb](https://gitlab.com/LinuxExperienceXYZ/lexweb)

Build command
hugo -v

Publish directory
dist

Deploy log visibility
Logs are public

Builds
Active
```

Both sites render and display fine locally.

**Comment by pacollins** [comment]
Did you ensure that Netlify is using Hugo Extended to build?

**Comment by tom-lex** [comment]
Thank you very much! I had checked that by ensuring the latest build image selection setting was selected (Ubuntu Xenial 16.04 image). According to netlify, this supports all Hugo versions.

However, your question pointed me in the right direction and I was able to adjust my [netlify.toml](https://gitlab.com/LinuxExperienceXYZ/lexweb/-/blob/master/netlify.toml) appropriately and build successfully. 

Thank you very much for taking the time and thank you for your work. Very much appreciated.
